modules/claims_api/lib/bgs_service/veteran_representative_service/read_all_veteran_representatives.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class VeteranRepresentativeService < ClaimsApi::LocalBGS
+    # type_code: form type (I.E. 21-22 vs 21-22A)
+    # ptcpnt_id: participant ID
+    def read_all_veteran_representatives(type_code:, ptcpnt_id:)
+      validate! type_code, ptcpnt_id
+
+      body = Nokogiri::XML::DocumentFragment.parse <<~EOXML
+        <ns0:CorpPtcpntIdFormTypeCode>
+          <veteranCorpPtcpntId>#{ptcpnt_id}</veteranCorpPtcpntId>
+          <formTypeCode>#{type_code}</formTypeCode>
+        </ns0:CorpPtcpntIdFormTypeCode>
+      EOXML
+      ret = make_request(action: 'readAllVeteranRepresentatives', body:)
+            &.dig('VeteranRepresentativeReturnList') || []
+      [ret].flatten
+    end
+
+    def read_all_veteran_representatives_thread(ptcpnt_id:, type_codes: %w[21-22 21-22A])
+      type_codes.each { validate! _1, ptcpnt_id }
+
+      threads = []
+      type_codes.each do |type_code|
+        threads << Thread.new do # rubocop:disable ThreadSafety/NewThread
```

Comment:
We usually try not to disable rubocop and instead try to fix the problem wherever possible. Are you modifying any shared resources? Can you think of any way to avoid this?

Reply:
`rubocop` will throw that anytime threading is used, the 'recommended' approach is `sidekiq`, but we (may) need this synchronously so I figure I'd make both calls parallel to speed it up. It's probably overkill since it's just 2 requests, but I figure any performance bumps help.

Reply:
ok, yeah, I think that's fine.